setup.py

```diff
@@ -569,15 +573,31 @@ def srcpath(name=None, suffix=".pyx", subdir="src"):
     include = data.get("include", [])
     include.append(numpy.get_include())
 
+    extra_comp_args = extra_compile_args.copy()
+    comp_macros = data.get("macros", macros)
+    undef_macros = []
+
+    if is_platform_zos():
+        language = data.get("language", None)
```

Comment:
`None` is already the default. Also, since you aren't using the language more than in the `if`, I'd simply write `if data.get('language') == 'c++':` and make this more compact.

This file is huge, so would prefer to not make this unnecessarily longer.

Reply:
Done, thanks!